axes/handlers/base.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any, Dict, Optional
+
+from django.http import HttpRequest
+
+
+class AxesBaseHandler:  # pylint: disable=unused-argument
```

Comment:
@aleksihakli I think this class could be [abstract](https://docs.python.org/3.7/library/abc.html ), seems to me that every specialized handler should implement all the methods defined in here. Also it would tell to developers clearly that this base class pretends to define the general API.

Reply:
That's a good idea, I'll see into the abstractness.